src/base/http/requestparser.cpp

```diff
@@ -45,53 +46,15 @@ namespace
 {
     const QByteArray EOH = QByteArray(CRLF).repeated(2);
 
+    using namespace Utils::ByteArray;
```

Comment:
shouldn't this be outside of namespace?

Reply:
I think there's no difference in this case. Or do you have any specific objections?

Reply:
There is an `using namespace Http;` above, those should be placed together IMO.

Reply:
Ok.